README.md

```diff
@@ -56,10 +55,56 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+# ############################################################################
+# Init cluster config for helm
+# ############################################################################
+
+data "ibm_container_cluster_config" "cluster_config" {
+  # Update this value with the ID or name of the cluster where the operators will be deployed
+  cluster_name_id = "cluster_id"
+}
+
+# ############################################################################
+# Config providers
+# ############################################################################
+
+provider "ibm" {
+  # Update this value with your IBM Cloud API key value
+  ibmcloud_api_key = "api key value"  # pragma: allowlist secret
+  # Update this value with the region that your cluster is deployed
+  region           = "us-south"
+}
+
+provider "helm" {
+  kubernetes {
+    host  = data.ibm_container_cluster_config.cluster_config.host
+    token = data.ibm_container_cluster_config.cluster_config.token
```

Comment:
add this: `cluster_ca_certificate = data.ibm_container_cluster_config.cluster_config.ca_certificate`